.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
```

Same for other workflows

---

Oops yes, sorry I copy-pasted from one of my other repos :sweat_smile: